test/e2e/specs/plans/plans__signup-business.ts

```diff
@@ -90,12 +90,8 @@ describe(
 				await startSiteFlow.clickButton( 'Skip to dashboard' );
 			} );
 
-			it( 'Skip Launchpad', async function () {
-				await page.waitForURL( /launchpad/, { waitUntil: 'networkidle' } );
-				await page.getByRole( 'button', { name: 'Skip for now' } ).click( { timeout: 20 * 1000 } );
-
-				// Launchpad redirects to `/home` when skipped.
-				await page.waitForURL( /home/ );
+			it( 'See Home', async function () {
+				await page.waitForURL( /home/, { waitUntil: 'networkidle' } );
```

Comment:
Quick suggestion here: I would take out the `networkidle` piece for this redirect -- it doesn't work as well on the home page because it has a lot of requests, and that might make the test a little more stable. If we land on the route and get the default document `load` event, that should be plenty good 👍

Reply:
Sounds good -- change looks great, thank you! 😄 

Did the rebase work cleanly for you though? It seems like the main fix is now missing in the diff now, but not sure if that was just a fluke on my end.

> But I could find the env.CALYPSO_BASE_URL option to update 🤔

It's a bit buried in the TeamCity UI. When you go to that "Parameters" tab in the TeamCity build settings (after clicking the three dots), you have to scroll a little bit!

Reply:
> Did the rebase work cleanly for you though? It seems like the main fix is now missing in the diff now, but not sure if that was just a fluke on my end.
The rebase messed up my changes. Sorry!

Reply:
Btw, just re-ran the pre-release tests for your changes, and they're running green! 🎉 🟢